lib/commons/dom/is-visible.js

```diff
@@ -96,6 +96,20 @@ function isAreaVisible(el, screenReader, recursed) {
   );
 }
 
+/**
+ * Check if a `<details>` element is visible.
+ * @param {HTMLDetailsElement} el The element to check
+ * @retruns {boolean}
+ */
+function isDetailsVisible(el) {
+  // IE11 does not support details (without a
+  // polyfill), so we need to check for the `open`
+  // property in the element to determine if
+  // the browser supports hiding content when
+  // the details element is closed
```

Comment:
This seems more than enough

```suggestion
  // Handle lack of support in IE
```